samples/local_attestation/localattestation.edl

```diff
@@ -7,23 +7,23 @@ enclave {
     from "platform.edl" import *;
 
     trusted {
-        public int get_target_info([out] uint8_t** target_info_buffer, [out] size_t* target_info_size);
+        public int get_enclave_format_settings([out] uint8_t** format_settings, [out] size_t* format_settings_size);
 
         // Return the public key of this enclave along with the enclave's report.
         // Another enclave can use the report to attest the enclave and verify
         // the integrity of the public key.
-        public int get_targeted_report_with_pubkey( [in, size=target_info_size] uint8_t* target_info_buffer,
+        public int get_targeted_evidence_with_pubkey( [in, size=target_info_size] uint8_t* target_info_buffer,
```

Comment:
Since you're renaming this function anyway, I will add:
OE coding style guidelines for public APIs and samples is to avoid abbreviations, and "pub" is an abbreviation.  This should be expanded to "public_key"